src/domain/proposal.ts

```diff
@@ -172,6 +174,10 @@ export function updateGPProposal(
     scheme.gpQueue = proposal.organizationId.toHex();
     scheme.save();
   }
+
+  let dao = getDAO(avatarAddress.toHex());
```

Comment:
this should be added to all proposals (not just one type)
how about adding that to the getProposal function